src/components/App.js

```diff
@@ -1,25 +1,27 @@
-import logo from '../assets/logo.svg';
 import '../styles/App.css';
+import { ChakraProvider } from '@chakra-ui/react';
+import { Container } from '@chakra-ui/react';
+import SignUp from './SignUp/SignUp';
+import LogIn from './LogIn/LogIn';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+	return (
```

Comment:
Not reviewing the PR, but interested to see how you did certain things. One thing I noticed is that the spacing is huge across files. Can we set the tab size to 2 spaces?

Reply:
can you be more specific regarding the spacing? Is it spacing inside the source code or the layout of the actual components?

Reply:
Spacing inside the source.
```html
<ChakraProvider>
	<Container maxWidth='container.xl' p={0}>
		<Router>
			<Switch>
				<Route path='/' exact>
					<SignUp />
				</Route>
				<Route path='/login' exact>
					<LogIn />
				</Route>
			</Switch>
		</Router>
	</Container>
</ChakraProvider>
```

can be condensed to 

```html
<ChakraProvider>
  <Container maxWidth='container.xl' p={0}>
    <Router>
      <Switch>
        <Route path='/' exact>
          <SignUp />
        </Route>
        <Route path='/login' exact>
          <LogIn />
        </Route>
      </Switch>
    </Router>
  </Container>
</ChakraProvider>
```

Reply:
I use prettier as well, but I am still able to set the tab size to 2 spaces.

Reply:
I agree to @zealptl . I think its the github cause in my source code the spacing is minimal